settings.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <title>Wandika - Account Settings</title>
+    <link rel="stylesheet" type="text/css" href="css/demo.css" />
+    <link rel="stylesheet" type="text/css" href="css/style.css" />
+    <script type="text/javascript" src="js/modernizr.custom.86080.js"></script>
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+</head>
+
+<body class="profile">
+    <div class="settings-container">
+        <header>
+            <div class="navBar">
+                <ul>
+                    <div class="logoBox">
+                        <li>
+                            <img class="logo" src="images\logo-wandika-word.png" alt="logo">
+                        </li>
+                    </div>
+
+                    <div class="menuElements">
+                        <li><a id="login" href="#">Login</a></li>
+                        <li><a id="signUp" href="#">Sign Up</a></li>
+                        <li><a href="contact.asp">Contact</a></li>
+                        <li><a href="about.asp">About</a></li>
+                    </div>
+
+                </ul>
+            </div>
```

Comment:
same here about the nav bar. we aren't copying and pasting. transition from the about page by using ajax calls to replace the content of the about page with the ones of the settings page